metastore/src/main/scala/com/gu/tableversions/metastore/VersionPaths.scala

```diff
@@ -35,4 +37,35 @@ object VersionPaths {
       new URI(versioned(normalised(partitionPath.toString)))
     }
 
+  /**
```

Comment:
These two functions are moved here from the SparkHiveMetastore but are useful for both metastore implementation. I though they fit in here